readthedocs/embed/v3/views.py

```diff
@@ -214,8 +214,19 @@ def _parse_based_on_doctool(self, page_content, fragment, doctool, doctoolversio
                     # </dl>
 
                     parent_node = node.parent
-                    if 'glossary' in node.parent.attributes.get('class'):
-                        next_node = node.next
+                    if "glossary" in node.parent.attributes.get("class"):
+                        # iterate through child and next nodes
+                        traverse = node.traverse()
+                        iteration = 0
+                        while iteration < 5:
+                            next_node = next(traverse, None)
+                            # TODO: Do we need to support terms with missing descriptions?
```

Comment:
Should we raise an error here? What is "not produce correct results"?

Reply:
I haven't tested this case. However, I think it's not possible to create a glossary entry without a description, tho